pytorch_pfn_extras/onnx/export_testcase.py

```diff
@@ -151,6 +155,7 @@ def _export(
         strip_large_tensor_data: bool = False,
         large_tensor_threshold: int = LARGE_TENSOR_DATA_THRESHOLD,
         use_pfto: bool = False,
+        return_output: bool = False,
```

Comment:
This should be true for compatibility with torch.onnx because they always returns `torch_out` except `torch.onnx.export`
https://sourcegraph.com/github.com/pytorch/pytorch@4bfb7341819b3bfcaf65ddc136f25d23983740a7/-/blob/torch/onnx/utils.py?L1663
https://sourcegraph.com/github.com/pytorch/pytorch@4bfb7341819b3bfcaf65ddc136f25d23983740a7/-/blob/torch/onnx/utils.py?L208

Reply:
Thanks, done!